code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -321,7 +321,7 @@
 /obj/item/borg/upgrade/selfrepair/action(mob/living/silicon/robot/R, user = usr)
 	. = ..()
 	if(.)
-		var/obj/item/borg/upgrade/selfrepair/U = locate() in R.module.modules
+		var/obj/item/borg/upgrade/selfrepair/U = locate() in R.contents
```

Comment:
You can just do in R, no need for in R.contents

Reply:
lazy